backend/app/Http/Controllers/StudentListController.php

```diff
@@ -19,4 +20,89 @@ public function studentList(Request $request)
 
         return view('student-list', compact('students', 'keyword', 'sortBy', 'sortOrder'));
     }
+    public function addStudent(Request $request)
+    {
+        return view('student-info');
+    }
+    public function  storeStudent(Request $request)
+    {
+        $student = new Student();
+        $student->name = $request->name;
+        $student->birthday = $request->birthday;
+        $student->save();
+
+        $gradeMath = new Grade();
+        $gradeMath->student_id = $student->id;
+        $gradeMath->subject = 'Math';
+        $gradeMath->grade = $request->math;
+        $gradeMath->save();
+
+        $gradeMusic = new Grade();
+        $gradeMusic->student_id = $student->id;
+        $gradeMusic->subject = 'Music';
+        $gradeMusic->grade = $request->music;
+        $gradeMusic->save();
+
+        $gradeEnglish = new Grade();
+        $gradeEnglish->student_id = $student->id;
+        $gradeEnglish->subject = 'English';
+        $gradeEnglish->grade = $request->english;
+        $gradeEnglish->save();
+
+        return redirect()->back()->with('success', 'Student added');
+    }
+    public function editStudent(Request $request, $id)
```

Comment:
@vinhhc97 
If `$request` isn't used, please remove it.
You can get `student` object when passing `id` or `object` of Student.
[Document at here](https://laravel.com/docs/7.x/routing#implicit-binding)
```suggestion
    public function editStudent(Student $student)
```